source/features/what-changed-since-previous-release.css

```diff
@@ -0,0 +1,14 @@
+.commit > ul > li.rgh-what-changed {
+	display: inline-block;
+	margin-top: 4px;
+	margin-right: 8px;
+}
+
+.release > div:first-child > ul > li.rgh-what-changed {
+	margin-bottom: 4px;
```

Comment:
See if you can replace these styles with GitHub's own margin classes, like `mb-1` and `d-inline-block`. Maybe you can apply them without any conditions (i.e. just use `className="mb-1 mr-1 mt-1"` on all)

Reply:
Much better. Everything just reduced to `d-inline-block mb-1 mt-1 f6`. 
Got rid of the `css` file.

Reply:
Don't forget to push your changes, I still see the file :)